sysbuild/Kconfig.pprcore

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SUPPORT_PPRCORE
+	bool
+	default y if SOC_NRF54H20_CPUAPP
+
+config PPRCORE_REMOTE_BOARD_TARGET_CPUCLUSTER
```

Comment:
Do we actually need this? PPR doesn't change name so maybe we can omit this.

Reply:
discussed on Teams